web/book/src/tutorial/aggregation.md

```diff
@@ -1,95 +1,98 @@
 # Aggregation
 
 A key feature of analytics is reducing many values down to some summary. This
-act is called aggregation and always includes an aggregation function, for
-example `average` or `sum`:
+act is called "aggregation" and always includes a function &mdash; for
+example, `average` or `sum` &mdash; that reduces the table to a single row.
 
-<!-- aggregating ids does not make much sense, should we switch to some other table? -->
+**aggregate:** The `aggregate` transform takes a tuple to create one or many new columns
+that "distill down" data from all the rows.
 
 ```
-from albums
-aggregate {sum_of_ids = sum album_id}
+from invoices
+aggregate { sum_of_orders = sum total }
```

Comment:
```suggestion
aggregate { sum_of_orders = sum amount }
```

using `total` is not the clearest name for the thing being summed...

Reply:
`total` is a column in `chinook/invoices`

Before the tutorial merges, I'd like to have actual tables with chinook data in the book, which would add a lot to understanding of PRQL.

Reply:
@aljazerzen is correct - the `invoices` table has a `total` column. Not my favorite name, either, but I wanted the examples to work in the playground.